schema/me/lot_standing.js

```diff
@@ -22,6 +25,13 @@ export const LotStandingType = new GraphQLObjectType({
     bidder: {
       type: Bidder.type,
     },
+    is_in_live_auction: {
```

Comment:
I don't think this is exactly what you want-- I think the important thing is whether or not the bid is on a sale that is currently _in_ live mode. (i.e. `live_start_at` is in the past). We can and should be showing the bid status for lots that will convert to live mode (just not when the live portion begins).

I also wonder, since we're now grabbing the `sale` here, why do we need this extra field? Seems just as easy to query for

```
me {
  lot_standings {
    sale {
      href
      is_live_open
    }
  }
}
```

(`is_live_open` already exists and has all of the logic we need-- wouldn't want to duplicate)

Reply:
Ah that makes sense!